drivers/gles3/shaders/scene.glsl

```diff
@@ -1073,7 +1073,7 @@ LIGHT_SHADER_CODE
 		float cNdotH = max(dot(N,H), 0.0);
 		float cLdotH = max(dot(L,H), 0.0);
 
-#if defined(LIGHT_USE_ANISOTROPY)
+# if defined(LIGHT_USE_ANISOTROPY)
```

Comment:
Was that changed automatically by your IDE/beautifier? All the C/C++ and GLSL code in Godot uses `#if` without space, so it's probably best to leave it that way until we decide otherwise for the whole code base.

Reply:
Nevermind, I just checked the context and this style is indeed used to show nesting, good for me :)